vulture/core.py

```diff
@@ -275,6 +277,9 @@ def report(self, min_confidence=0, sort_by_size=False):
             self.found_dead_code_or_error = True
         return self.found_dead_code_or_error
 
+    def _ignore_name(self, name):
```

Comment:
This can become a local function in `_define`.

Reply:
Since, we don't use `_define` for decorators, I think we would need to keep it separate, so that it can still be used to ignore decorated functions.

Reply:
You're right.